ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/sles.yml

```diff
@@ -2,14 +2,16 @@
 - name: Add Docker Repo for SLES15
   command: zypper ar https://download.docker.com/linux/sles/docker-ce.repo
   when:
+    - ansible_distribution_major_version == "15"
     - ansible_architecture == "x86_64" or ansible_architecture == "s390x"
 
-- name: Add security repo for container-selinux
-  community.general.zypper_repository:
+- name: Add security repo for container-selinux for SLES 15
+  zypper_repository:
```

Comment:
What's the reason for removing the full qualification here? Did you find it didn't work?

Reply:
Full qualification was not working. This is the error I got:
```
couldn't resolve module/action 'community.general.zypper_repository`
```

I searched within the entire Adoptium PB. The same module is used in the `Common` role without the full name.
https://github.com/adoptium/infrastructure/blob/0077e292c4691e2f477f361d6535382dda938363/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/SLES.yml#L9-L11

And also in the documentation for Ansible 2.9, it's also using `zypper_repository`. And after changing it to just the module name, it worked fine.

And to add this, the security repo for container-selinux for SLES 15 is no longer valid. 
The link doesn't exist anymore: https://download.opensuse.org/repositories/security:SELinux/SLE_15_SP3/security:SELinux.repo

Reply:
Thanks for the explanation on the qualification of `zypper_repository`

> the security repo for container-selinux for SLES 15 is no longer valid.

FYI @Haroon-Khel - any thoughts on this?